packages/uniforms-material/src/SelectField.tsx

```diff
@@ -126,7 +127,7 @@ function Select(props: SelectFieldProps) {
                   name={name}
                   // FIXME: There's a problem with SelectFieldProps.
                   onChange={() => disabled || onChange!(xor(item, value))}
-                  ref={inputRef}
+                  inputRef={inputRef}
```

Comment:
It should stay as `ref`. `inputRef` is our internal name for forwarded ref.

Now there's `forwardRef` for that and we should consider using it. Anyway, it'll be a separate PR.